README.md

```diff
@@ -270,12 +270,33 @@ If you would like to contribute to the translation of Open Event, you need to [r
 
 Once you have activated your account just proceed to the [translation section](https://hosted.weblate.org/projects/open-event/).
 
+## Pre-commits guide
 
-## Contributions, Bug Reports, Feature Requests
+**Introduction**
+
+Git hook scripts are useful for identifying simple issues before submission to code review. We run our hooks on every commit to automatically point out issues in code such as missing semicolons, trailing whitespace, and debug statements. By pointing these issues out before code review, this allows a code reviewer to focus on the architecture of a change while not wasting time with trivial style nitpicks.
+
+### Quick Start
+
+ #### 1. Add a pre-commit configuration ¶
+
+* create a file named .pre-commit-config.yaml
+* you can generate a very basic configuration using pre-commit sample-config
+
+ #### 2. Install the git hook scripts ¶
+* run pre-commit install to set up the git hook scripts
+```sh
+$ pre-commit install
+pre-commit installed at .git/hooks/pre-commit
+```
+* now pre-commit will run automatically on git commit!
```

Comment:
You need to document about `configuring`. This is for Setup, which is already there.

Reply:
Even for documenting configuration, I think pasting the link `https://pre-commit.com` is enough. @iamareebjamal What do you think?

Reply:
We should still document `pre-commit install`